cmd/controller/controller.go

```diff
@@ -32,17 +32,25 @@ import (
 )
 
 const (
-	NDMKind        = "Disk"                   // NDMKind is the CR kind.
-	NDMVersion     = "openebs.io/v1alpha1"    // NDMVersion is the CR version.
-	NDMHostKey     = "kubernetes.io/hostname" // NDMHostKey is host name label prefix.
-	NDMActive      = "Active"                 // NDMActive is constant for active resource status.
-	NDMInactive    = "Inactive"               // NDMInactive is constant for inactive resource status.
-	NDMUnknown     = "Unknown"                // NDMUnknown is constant for resource unknown satus.
-	NDMDiskTypeKey = "ndm.io/disk-type"       // NDMDiskTypeKey specifies the type of disk
+	// NDMKind is the CR kind.
+	NDMKind        = "Disk"        
```

Comment:
<!--Since you have put the comments on the top of every vaiable (which is good), so these constants no longer needs to maintain `=` at same level. So please remove any extra spaces before `=`, just keep one space before and one space after the equal sign. Or even better, j-->Just run `go fmt cmd/controller/controller.go` and update the PR.

Reply:
@a4abhishek  I am not familiar with go ,so can you tell me the steps to do so .

Reply:
Sure, we'll talk on slack.